include/deal.II/base/function_lib.h

```diff
@@ -1254,6 +1254,81 @@ namespace Functions
      */
     const Table<dim,double>                     data_values;
   };
+
+
+  /**
+   * A class that represents a function object for a polynomial. A polynomial
+   * is composed by multiple monomials,i-e. in 1-d they have the form
+   * $x^\alpha + x^\beta + ... + x^\epsilon$, in 2-d the form
```

Comment:
but polynomials generally have coefficients in front of the `x^alpha` terms, no?